src/Config/Services.php

```diff
@@ -31,7 +32,12 @@ public static function renderer(?string $viewPath = null, ?ViewConfig $config =
             return static::getSharedInstance('renderer', $viewPath, $config);
         }
 
-        $viewPath = $viewPath ?: config('Paths')->viewDirectory;
+        if ($viewPath === null && ENVIRONMENT === 'testing') {
```

Comment:
We can't have it here. This will cause the `$viewPath` to be changed when you start writing tests for your app.